File: module_wan/src/main/java/com/huyingbao/module/wan/ui/friend/adapter/WebSiteAdapter.java
package com.huyingbao.module.wan.ui.friend.adapter;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.BaseViewHolder;
import com.huyingbao.module.wan.R;
import com.huyingbao.module.wan.ui.friend.model.WebSite;

import java.util.List;

import androidx.annotation.Nullable;

/**
 * @author liujunfeng
 * @date 2019/1/1
 */
public class WebSiteAdapter extends BaseQuickAdapter<WebSite, BaseViewHolder> {
    public WebSiteAdapter(@Nullable List<WebSite> data) {
        super(R.layout.wan_recycle_item, data);
    }

    @Override
    protected void convert(BaseViewHolder helper, WebSite item) {
        helper.setText(R.id.tv_item_name, item.getName())
                .setText(R.id.tv_item_description, item.getLink())
                .setText(R.id.tv_item_id, "GithubId" + item.getId());
    }
}

File: module_wan_kotlin/src/main/java/com/huyingbao/module/wan/ui/friend/adapter/WebSiteAdapter.kt
package com.huyingbao.module.wan.ui.friend.adapter

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.BaseViewHolder
import com.huyingbao.module.wan.R
import com.huyingbao.module.wan.ui.friend.model.WebSite

/**
 * Created by liujunfeng on 2019/1/1.
 */
class WebSiteAdapter(data: List<WebSite>?) : BaseQuickAdapter<WebSite, BaseViewHolder>(R.layout.wan_recycle_item, data) {

    override fun convert(helper: BaseViewHolder, item: WebSite) {
        helper.setText(R.id.tv_item_name, item.name)
                .setText(R.id.tv_item_description, item.link)
                .setText(R.id.tv_item_id, "GithubId" + item.id)
    }
}
